backend/EasyMeets.Core/EasyMeets.Core.BLL/Services/AvailabilityService.cs

```diff
@@ -17,14 +18,19 @@ public class AvailabilityService : BaseService, IAvailabilityService
 
         public async Task<UserPersonalAndTeamSlotsDto> GetUserPersonalAndTeamSlotsAsync(long id)
         {
-            var availabilitySlots = await _context.AvailabilitySlots
+            var availabilitySlots = (await _context.AvailabilitySlots
                 .Include(x => x.Members)
                     .ThenInclude(x => x.User)
                 .Include(x => x.Author)
                 .Include(x => x.Team)
-                .Where(x => x.CreatedBy == id && x.Members.Any(x => x.UserId == id))
+                .Include(x => x.Schedule)
+                    .ThenInclude(s => s.ScheduleItems)
+                .Where(x => x.CreatedBy == id || x.Members.Any(x => x.UserId == id))
+                .ToListAsync())
```

Comment:
Why did you remove the projection?

Why do we need Schedule here?

Reply:
We need schedule here to be able to edit it using data from this dto

Reply:
I don't think so
This method is for loading all slots, not for some specific

Reply:
Still it loads all the slot's data except for schedule. We might need schedule not only for editing in the future, for example displaying slot for external booking